### PR TITLE
[PPP-4865] - Vulnerable component Snakeyaml - Fixing problem with transient dependencies

### PR DESCRIPTION
**Pull request by andreramos89**
@smmribeiro 
@renato-s 

Fixing problem with transient dependencies

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 28m 12s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **898**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/big-data-hadoop-shims/job/PR-1397/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: shims/hdp30/driver/pom.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
     <!-- Note the difference between \lib and \lib\client -->
     <jetty.lib-folder.version>9.3.20.v20170531</jetty.lib-folder.version>
     <jetty.client-folder.version>9.3.19.v20170502</jetty.client-folder.version>
-
+    <safeyaml.version>1.34.1</safeyaml.version>
```
Comment (smmribeiro) [inline shims/hdp30/driver/pom.xml:55]: Is this version necessary here?

**Review by smmribeiro — APPROVED** [review]
Looks good.

File: shims/hdp30/driver/pom.xml (hunk 1 above)
Comment (andreramos89) [reply shims/hdp30/driver/pom.xml:55]: Yes, apparently

**Comment by andreramos89** [comment]
Please note that these changes were necessary because on packaging the dependencies, as snakeyaml was replaced with Safeyaml, maven was packaging older versions of this library.

**Pull request merged**